pygmt/tests/test_makecpt.py

```diff
@@ -57,25 +48,25 @@ def test_makecpt_plot_points(points, region):
 
 
 @pytest.mark.mpl_image_compare
-def test_makecpt_plot_grid(grid):
+def test_makecpt_plot_grid(region):
     """
     Use static color palette table to change color of grid.
     """
     fig = Figure()
     makecpt(cmap="relief")
-    fig.grdimage(grid, projection="W0/10c")
+    fig.colorbar(cmap=True, region=region, frame=True, position="JBC")
```

Comment:
Is `region` required here?

Reply:
I get this error `colorbar [ERROR]: Option -DJ requires the -R option` when I only use `fig.colorbar(cmap=True, frame=True, position="JBC")`

Reply:
Yes, `-DJ` (or `position="JBC"`) needs a `region`. For internal tests, maybe `fig.colorbar(position="x0c/0c+w10c")` is enough.